### PR TITLE
Updated docs for M1, added port mapping for db and winged-keys db

### PR DESCRIPTION
**Pull request by vonbearshark**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ services:
       WINGED_KEYS_HOST: 'https://winged-keys:5050'
       # Change this to true if you want to create a test user with permissions for multiple organizations
       MULTI_ORG_USER: 'false'
+      PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 'true'
```
Comment (vonbearshark) [inline docker-compose.yaml:36]: Not sure if this is an issue. If so, happy to just add it to the `README` instead, with the db `image` update

Comment (jlhogan) [reply docker-compose.yaml:36]: im fine with this :)

File: docker-compose.yaml [hunk 2]
```diff
@@ -41,12 +42,16 @@ services:
     environment:
       SA_PASSWORD: TestPassword1
       ACCEPT_EULA: Y
+    ports:
```
Comment (jlhogan) [inline docker-compose.yaml:45]: why do you want the DB ports exposed? no issue, just curious. are u using a GUI for the DB stuff? which one?

**Review by jlhogan — APPROVED** [review]

File: docker-compose.yaml (hunk 2 above)
Comment (vonbearshark) [reply docker-compose.yaml:45]: Yep, exactly. I use [TablePlus](https://tableplus.com/). It is fine, pretty simple/intuitive, which is my only standard

**Review by bamader — APPROVED** [review]

**Pull request merged**